utils.py

```diff
@@ -160,7 +160,7 @@ def downloadSourceTarball(output_dir="./packages/"):
     if [[ $os != "AIX" && $os != "Darwin" && $os != "FreeBSD" ]]; then
       use_flock="flock \"$d$f\" "
     fi
-    cmd="$use_flock wget -cO \"$d$f\" \""""
+    cmd="$use_flock wget --progress=bar:force:noscroll -cO \"$d$f\" \""""
```

Comment:
Do you know if this looks good in buildbot? Since BB does basically a line-by-line log gathering I am not sure if the progress bar wouldn't be re-written for each line. Anyway something to test out.

Reply:
it looks not good but better, instead of dot, it will print `=>`, I tested it but since I reset the stack, it's not available anymore (see link). As said, depending of the bandwidth, it reduce the output from ~2000 lines to ~150.